bundles/org.openhab.binding.homematic/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -95,6 +95,18 @@
 				<advanced>true</advanced>
 				<default>9292</default>
 			</parameter>
+			<parameter name="callbackRegistrationRetries" type="integer" min="1">
+				<label>Number of Callback Registration Retries </label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Also there seems to be a space at the end that should be removed IMO

Reply:
OK, will find a shorter label name.

Reply:
I have changed the configuration and this parameter has been removed